### ISSUE TITLE
Incorrect focus state jump

### ISSUE THREAD

**Issue by srtalbot** [cds-snc/platform-forms-client#295]
When a required field is not filled in the focus state jumps to the last missing field. Below: two fields are missing, name, and email. The focus state brings the user to the email field

![image](https://user-images.githubusercontent.com/7973770/125324231-b96de880-e30d-11eb-8a35-a597609b561d.png)

@brdunfield , can you correct me if I'm wrong from an accessibility perspective, but we should have the focus state bringing people to the "Please correct the errors on the page" heading. 

![image](https://user-images.githubusercontent.com/7973770/125324422-e6220000-e30d-11eb-9e86-9ce7cfdd0a33.png)

**Comment by bryan-robitaille** [comment]
See issue #323.
